critiquebrainz/db/review.py

```diff
@@ -487,12 +487,18 @@ def get_reviews_list(connection, *, inc_drafts=False, inc_hidden=False, entity_i
                 "created": row.pop("user_created"),
             })
 
+        if review_type == 'rating':
```

Comment:
sorry - looking back at my original comment I realise that I wasn't very clear here.
What I meant to say by moving this parameter into `get_reviews_list`, is that we should perform this filtering in the SQL query above (by adding items to `filters`/`filter_data`), instead of performing a post-filtering in python.